LectureNotes/Week-04/readme.md

```diff
@@ -0,0 +1,12 @@
+# Week 4 - Password Validation and Salted Hashing
+### Application Security And Hardening
+
+[Return Home](../../../../)  
+
+[View Lecture Notes](http://coreyshuman.github.io/GeekwiseApplicationSecurity/LectureNotes/Week-04)  
+
+### Goals
```

Comment:
Goals:
- Add basic user management to our application
- Use Bcrypt to enable password salting and hashing
- Investigate different types of user authorization (tokens, cookies, etc)